### ISSUE TITLE
Improve editor accesibility with screen readers

### ISSUE THREAD

**Issue by brunoprietog** [mb21/panwriter#40]
Hello!
Pandoc is very useful for blind people, since it generates accessible documents using HTML, docx, epub output, among others, excluding PDF. 
This application has a very big potential, especially for those who do not handle the command line. However, the editor is not accessible with screen readers. All the rest, including the buttons and the menu bar reads very well, since everything is correctly labeled.
To fix this I suggest using the [Monaco Editor](https://github.com/Microsoft/monaco-editor/), which provides many accessibility options.
thanks!

**Comment by mb21** [comment]
Thanks for the feedback!
PanWriter is usind CodeMirror 5... didn't know that that wasn't accessible... :-( switching to Monaco or CodeMirror 6 (once it's released) is definitely something we should consider... but it's not going to be a quick fix...

**Comment by brunoprietog** [comment]
Thanks a lot for the answer, I just realized that CodeMirror 6 brings accessibility improvements. I hope you can implement either option. It would be interesting if you could comment here if you make any progress on this.
Greetings and thanks again, I'll keep an eye out

**Comment by ccdavis** [comment]
Putting in a vote for screen reader accessibility. Would love to try it when you  got to Codemirror 6.
